app/waf/test/acceptance/application.controller.acceptance.ts

```diff
@@ -176,6 +176,7 @@ describe('ApplicationController', () => {
         adcId: adc.id,
       });
       let pool = await givenPoolData(wafapp, {
+        id: uuid(),
```

Comment:
I see createPoolObjectWithID() already can generate a uuid, why we must input one here?